cmd/fpga_plugin/fpga_plugin.go

```diff
@@ -170,10 +205,18 @@ func (dm *deviceManager) PreStartContainer(ctx context.Context, rqt *pluginapi.P
 }
 
 func main() {
+	var mode string
+
+	flag.StringVar(&mode, "mode", "region", "device plugin mode: 'region' (default) or 'af'")
 	flag.Parse()
-	fmt.Println("FPGA device plugin started")
 
-	devs, err := discoverFPGAs(sysfsDirectory, devfsDirectory)
+	if mode != "region" && mode != "af" {
+		glog.Fatal("Wrong mode: ", mode)
```

Comment:
modes better to be declared as constants (so it can be used instead of explicit strings in multiple comparisons), and maybe it will make sense to have small validation function, if we going to have more than two modes.